lib/store.js

```diff
@@ -107,10 +109,21 @@ var storeClient = (function() {
         );
     };
 
+    var validateUsage = function(usageInfo, userInfo, callback)  {
```

Comment:
If you know that `userInfo` is always `null`, you can skip this parameter so callers do not have to include it.